test/unit/presenters/api/integrations_show_presenter_test.rb

```diff
@@ -49,18 +49,6 @@ def test_environments_have_same_config?
     assert Presenter.new(@proxy).environments_have_same_config?
   end
 
-  test 'state_modifier' do
-    @proxy.account.stubs(:provider_can_use?).with(:api_as_product).returns(true)
-
-    assert_equal 'is-untested', Presenter.new(@proxy).test_state_modifier
-
-    @proxy.api_test_success = true
```

Comment:
Does this mean that we can eventually also remove the field `api_test_success` or do we still use it for something?
If we can remove it, plz add a `TODO` comment (and probably a Jira).

Reply:
It is still used in other places, for instance https://github.com/3scale/porta/blob/e9e71382e25b2a8a43f3d496c793e72117a92c23/test%2Funit%2Fproxy_test.rb#L512